.github/workflows/acceptance-test-flaky-tests.yml

```diff
@@ -0,0 +1,531 @@
+# This GitHub action runs your tests for each commit push and/or PR. Optionally
+# you can turn it on using a cron schedule for regular testing.
+#
+# This pipeline is for isolating the known flaky tests until they can be properly
+# triaged and corrected through API improvements, test improvements, and so on.
+name: Failing Acceptance Tests (Feature Flagged)
```

Comment:
I'd probably change this name to indicate that this is multi-region, and remove "Feature Flagged"

Reply:
yes, that's what I get for copying & pasting